WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -545,8 +548,10 @@ public void onClick(DialogInterface dialog, int id) {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
-        MenuInflater inflater = getMenuInflater();
-        inflater.inflate(R.menu.stats, menu);
+        if (!mNoMenuDrawer) {
```

Comment:
> I think this is not needed here, otherwise the "View full site" will not be available.

Ok I thought we wanted the user to stay on that level only (and not go deeper). I'll remove that.